src/App.tsx

```diff
@@ -7,56 +7,149 @@ import classNames from 'classnames';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
+import { User } from './types/User';
+import { getPostsByUser, getUsers, postInfo } from './api/posts';
+import { Post } from './types/Post';
 import { Loader } from './components/Loader';
+import { Comment } from './types/Comment';
 
 export const App: React.FC = () => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [users, setUsers] = useState<User[]>([]);
+  const [currentUserPosts, setcurrentUserPosts] = useState<Post[]>([]);
+  const [error, setError] = useState('');
+  const [showLoader, setShowLoader] = useState(false);
+  const [isSideBarIsAlreadyOpen, setIsSideBarIsAlreadyOpen] = useState(false);
```

Comment:
Consider renaming the state variable 'isSideBarIsAlreadyOpen' to 'isSidebarOpen' for better readability.